icepyx/core/read.py

```diff
@@ -5,7 +5,6 @@
 
 import earthaccess
 import numpy as np
-from s3fs.core import S3File
```

Comment:
Can we comment this one out for now instead of removing? The linter is catching it because the line of code is commented (line 637), but ultimately I'd like to have better handling there that makes sure the file gets closed.

Reply:
I'm cool with that. What do you think of including the commented import with the commented code? Then it's obvious to a reader not to erase the commented import. And when we un-comment the code, our linter will ensure we put the import in the right place!

Reply:
Yep, commenting that line and moving it down below sounds good, can you apply that change please?

Reply:
Can do!

Reply:
ad95e2e